### PR TITLE
Add mcafee_antivirus_definitions_updated OVAL and XCCDF variable

### PR DESCRIPTION
**Pull request by redhatrises**
- Add mcafee_antivirus_definitions_updated OVAL and XCCDF variable
- Set the RHEL7 DISA profile to 1 week

### CODE REVIEW

File: shared/checks/oval/mcafee_antivirus_definitions_updated.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<def-group oval_version="5.10">
+  <definition class="compliance" id="mcafee_antivirus_definitions_updated" version="1">
+    <metadata>
+      <title>Confirm Existence and Permissions of System Log Files</title>
```
Comment (yuumasato) [inline shared/checks/oval/mcafee_antivirus_definitions_updated.xml]: Title of this check is not related to antiviruses.

File: shared/checks/oval/mcafee_antivirus_definitions_updated.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  <local_variable id="variable_mcafee_dat_files_mtime" version="1" comment="Age of file for each McAfee definition file." datatype="int">
+      <time_difference format_2="seconds_since_epoch">
+        <object_component object_ref="mcafee_dat_files_mtime" item_field="m_time"/> 
+      </time_difference>
+  </local_variable>
+
+  <ind:variable_test check="all" check_existence="all_exist" id="test_mcafee_antivirus_definitions_updated" version="1" comment="McAfee AntiVirus definitions have been updated">
+    <ind:object object_ref="object_mcafee_antivirus_definitions_updated" />
+    <ind:state state_ref="state_mcafee_antivirus_definitions_updated" />
```
Comment (yuumasato) [inline shared/checks/oval/mcafee_antivirus_definitions_updated.xml]: Are the objects and states always named after the name of tests?

Nitpick:
Would be more clear if this object was called something like "antivirus definitions modified time".
And state called "antivirus definition timeout", or "max modified time".

File: shared/checks/oval/mcafee_antivirus_definitions_updated.xml (hunk 1 above)
Comment (redhatrises) [reply shared/checks/oval/mcafee_antivirus_definitions_updated.xml]: hahaha... thanks @yuumasato

File: shared/checks/oval/mcafee_antivirus_definitions_updated.xml (hunk 2 above)
Comment (redhatrises) [reply shared/checks/oval/mcafee_antivirus_definitions_updated.xml]: > Are the objects and states always named after the name of tests?

They are done more often than not, but updated them per your suggestion.

**Comment by yuumasato** [comment]
Ack.
